src/utils/mergeRefs.ts

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+/**
+ * Merge all refs into one ref function
+ * @param refs - list of refs to be merged (skip undefined)
+ */
+export function mergeRefs<T>(
```

Comment:
maybe should add some simple test to this function

Reply:
agree!